packages/eas-cli/src/project/expoConfig.ts

```diff
@@ -40,12 +59,38 @@ function getExpoConfigInternal(
 }
 
 export function getPrivateExpoConfig(projectDir: string, opts: ExpoConfigOptions = {}): ExpoConfig {
+  ensureExpoConfigExists(projectDir);
+
   return getExpoConfigInternal(projectDir, { ...opts, isPublicConfig: false });
 }
 
+export function ensureExpoConfigExists(projectDir: string): void {
+  const paths = getConfigFilePaths(projectDir);
+  if (!paths?.staticConfigPath && !paths?.dynamicConfigPath) {
+    // eslint-disable-next-line node/no-sync
+    fs.writeFileSync(path.join(projectDir, 'app.json'), JSON.stringify({ expo: {} }, null, 2));
+  }
+}
+
+async function ensureStaticExpoConfigIsValidAsync(projectDir: string): Promise<void> {
+  const paths = getConfigFilePaths(projectDir);
+  if (paths?.staticConfigPath?.endsWith('app.json')) {
```

Comment:
Is it possible for someone to not have a static config path at all, and opt to do all their configuration with a dynamic config path? If so, then we will need to do something like this:
```suggestion
  if (paths?.staticConfigPath?.endsWith('app.json') && !paths?.dynamicConfigPath) {
```